### ISSUE TITLE
create subset of data for debug on cpu

### ISSUE THREAD

**Issue by smilesun** [compstat-lmu/paper_2019_variationalResampleDistributionShift#13]
```
X, y = utils_parent.load_mnist("fashion-mnist")
from sklearn.model_selection import train_test_split
X_1, X_2, Y_1, Y_2 = train_test_split(X, y, stratify=y, test_size=0.01)
```

now X_2, Y_2 only have 70 pictures per class label.
